docs/config/01-configuration-file.md

```diff
@@ -793,6 +793,20 @@ If set then the following fields will be defined and can be overriden:
 All of Karma's urls get prefixed with the `urlRoot`. This is helpful when using proxies, as
 sometimes you might want to proxy a url that is already taken by Karma.
 
+## browserSocketTimeout
+**Type:** Number
+
+**Default:** `2000`
+
+**Description:** Timeout for the client socket connection (in ms).
+
+This configuration represents the amount of time that the client will wait for the socket
+to connect.
+
+When running a browser in different environments, it can take different amounts of time for the
+client socket to connect. If Karma cannot connect within the default timeout, you
```

Comment:
Please add a sentence explaining what a user will see in the case that the timeout is too small.

Reply:
Done!